docs/reference/configuration/network-mapper/kafka-watcher.mdx

```diff
@@ -0,0 +1,159 @@
+---
+sidebar_position: 2
+title: Kafka Watcher
+---
+
+To deploy the network mapper with the Kafka watcher component, do the following:
+```bash
+helm repo add otterize https://helm.otterize.com
+helm repo update
+helm install network-mapper otterize/network-mapper -n otterize-system --create-namespace --set kafkawatcher.enable=true
+```
+Make sure to include `--set kafkaServers={}` with a list of Kafka servers the Kafka watcher
+should examine logs for.
```

Comment:
-> "... with a list of Kafka servers whose logs the Kafka watcher should watch."